packages/playwright-ct-react/registerSource.mjs

```diff
@@ -69,10 +70,13 @@ function render(component) {
 }
 
 window.playwrightMount = async (component, rootElement, hooksConfig) => {
-  for (const hook of /** @type {any} */(window).__pw_hooks_before_mount || [])
-    await hook({ hooksConfig });
+  let App = () => render(component);
+  for (const hook of /** @type {any} */(window).__pw_hooks_before_mount || []) {
+    const wrapper = await hook({ App, hooksConfig });
+    if (wrapper) App = () => wrapper;
```

Comment:
nit: style, if expression goes to the next line.

Reply:
fixed